_inc/lib/plugins.php

```diff
@@ -66,6 +100,12 @@ protected static function generate_wordpress_org_plugin_download_link( $plugin_s
 	 }
 
 	 public static function get_plugin_id_by_slug( $slug ) {
+		// Check if get_plugins() function exists. This is required on the front end of the
```

Comment:
We're also not in the front end of the site, are we?

Reply:
Indeed. I was copy-pasting this from elsewhere. Since #8464 worked without it, I guess I can just frop it?

Reply:
I'd prefer keeping it just in case, because those admin files are often not specifically loaded. We can consider removing it only if we're sure that in all cases this method is called, the file is specifically required.

Reply:
Nope, has to remain for our use case:

```
[08-Jan-2018 15:54:56 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_plugins() in ~/public_html/wp-content/plugins/jetpack/_inc/lib/plugins.php:104
```